index.js

```diff
@@ -233,7 +238,7 @@ NYC.prototype.walkAllFiles = function (dir, visitor) {
 }
 
 NYC.prototype._maybeInstrumentSource = function (code, filename, relFile) {
-  var instrument = this.exclude.shouldInstrument(filename, relFile)
+  var instrument = (!this.instrument && this.all) || this.exclude.shouldInstrument(filename, relFile)
```

Comment:
Yeah, I mentioned this above in a comment:
> I can imagine, however, still wanting to limit that crawl to a known location. If my sources are in `src/` and my transpiled+instrumented files are in `dist/`, I want to be able to say so via something like `include` and `exclude`, rather than let `nyc` loose on my entire directory tree.